server.js

```diff
@@ -22,10 +22,12 @@ app.use(express.static("website"));
 // Setup Server
 app.listen(3000, () => console.log("Listening on port 3000..."));
 
+// TODO: what does data-object mean?: https://restfulapi.net/resource-naming/
 app.get("/data-object", (req, res) => {
```

Comment:
As said on the TODO, **REST API** endpoints should be more clear what they do, the name should be something related to **weather** or so, you can check this resource: https://restfulapi.net/resource-naming/